daffodil-core/src/main/scala/org/apache/daffodil/runtime1/ElementBaseRuntime1Mixin.scala

```diff
@@ -81,6 +81,9 @@ trait ElementBaseRuntime1Mixin { self: ElementBase =>
 
     // For simple elements with text representation, valueLength is captured in
     // individual parsers since they handle removing delimiters and padding.
```

Comment:
This comment is ambiguous to me. 

"Capture in individual parsers" to me means "the separate capturing parsers", but we mean the opposite of that. 

How about "Capture directly by the value parsers".

Reply:
makes sense, will fix